debounce-batch.js

```diff
@@ -63,10 +64,10 @@ module.exports = class DebouncingBatchAdd {
     }, this.period * 0.5)
   }
 
-  add(msgVal, cb) {
+  add(msgVal, opts, cb) {
```

Comment:
this function needs to be aware of buttwoo's author + parent type

Reply:
You need to bucket them into correct chains (so previous matches) and right now we don't, meaning that if you have parent on a message then it will be put into the same bucket as one without and validation will fail with wrong previous. There is a test in https://github.com/ssbc/ssb-ebt/pull/68 that fails because of this :)

Reply:
Ooh, interesting

Reply:
I can push a commit soon that fixes this.